web/app/js/components/JaegerLink.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import _isEmpty from 'lodash/isEmpty';
+import { jaegerIcon } from './util/SvgWrappers.jsx';
+
+const JaegerLink = ({ PrefixedLink, name, namespace, resource }) => {
+  let link = '/jaeger/search?service=linkerd-proxy&tags=';
+  if (_isEmpty(namespace)) {
```

Comment:
Minor optimization again, but WDYT about moving this into a function? I'm thinking that `to={baseLink + getQuery()` is a little more readable.

Reply:
done!